### PR TITLE
Expression bodies for query test base classes

### PR DESCRIPTION
**Pull request by ajcvickers**
And other code cleanup that happened at the same time.

First part of #26760

### CODE REVIEW

File: test/EFCore.Specification.Tests/Query/Ef6GroupByTestBase.cs [hunk 1]
```diff
@@ -1562,7 +1473,21 @@ private static IReadOnlyList<Person> CreatePeople()
                         LastName = "Don",
                         Age = 21,
                         Feet = new Feet { Id = 7, Size = 12 },
-                        Shoes = { new() { Id = 13, Style = "Sneakers", Age = 19 }, new() { Id = 14, Style = "Dress", Age = 20 } }
+                        Shoes =
+                        {
+                            new Shoes
```
Comment (roji) [inline test/EFCore.Specification.Tests/Query/Ef6GroupByTestBase.cs:1478]: Maybe leave this on single line, as long as it doesn't go over 120 I think cleanup leaves that alone

**Review by roji — APPROVED** [review]
So modern

File: test/EFCore.Specification.Tests/Query/Ef6GroupByTestBase.cs (hunk 1 above)
Comment (ajcvickers) [reply test/EFCore.Specification.Tests/Query/Ef6GroupByTestBase.cs:1478]: There's nothing manual in this PR--it's all automatic cleanup. I can try to disable/change this rule if people think that's the right way to go.

**Comment by ajcvickers** [comment]
Would like at least one of @smitpatel and @maumar to take a look before I go forward. (You're the ones who will be affected the most by this code. 😄) (Also, I still remember that time I got called into the Dev Lead's office and got a talking to when I merged code in their area without them signing off. 😉)

File: test/EFCore.Specification.Tests/Query/Ef6GroupByTestBase.cs (hunk 1 above)
Comment (roji) [reply test/EFCore.Specification.Tests/Query/Ef6GroupByTestBase.cs:1478]: No problem for merging as-is from my side

**Review by maumar — APPROVED** [review]

**Comment by maumar** [comment]
no hiccups on my end either :shipit:

**Pull request merged**